### PR TITLE
Implement "Use existing maps" feature in Missing Maps section

### PR DESCRIPTION
**Pull request by RZR-UA**

### CODE REVIEW

**Comment by RZR-UA** [comment]
Issue: https://github.com/osmandapp/OsmAnd/issues/19982

Android request to compare: https://github.com/osmandapp/OsmAnd/pull/20171

File: Sources/Controllers/Resources/OARequiredMapsResourceViewController/OARequiredMapsResourceViewController.mm [hunk 1]
```diff
@@ -277,7 +290,16 @@ - (UITableViewCell *)getRow:(NSIndexPath *)indexPath
             [cell.button setTitleColor:[UIColor colorNamed:ACColorNameTextColorActive] forState:UIControlStateHighlighted];
             cell.button.tag = indexPath.section << 10 | indexPath.row;
             [cell.button removeTarget:self action:NULL forControlEvents:UIControlEventTouchUpInside];
-            [cell.button addTarget:self action:@selector(onCalculateOnlineButtonPressed:) forControlEvents:UIControlEventTouchUpInside];
+
+            if ([item.title isEqualToString:OALocalizedString(@"calculate_online")])
```
Comment (alex-osm) [inline Sources/Controllers/Resources/OARequiredMapsResourceViewController/OARequiredMapsResourceViewController.mm:294]: It is bad practice to use localized title to distinguish items. Use `key` property instead.

File: Sources/Controllers/Resources/OARequiredMapsResourceViewController/OARequiredMapsResourceViewController.mm [hunk 2]
```diff
@@ -479,6 +501,14 @@ - (void)selectDeselectItem:(NSIndexPath *)indexPath
     [self setupNavbarButtons];
 }
 
+- (void)onIgnoreMissingMapsButtonPressed:(id)sender
+{
+    NSLog(@"onIgnoreMissingMapsButtonPressed");
```
Comment (alex-osm) [inline Sources/Controllers/Resources/OARequiredMapsResourceViewController/OARequiredMapsResourceViewController.mm:506]: Drop log

File: Sources/Controllers/Resources/OARequiredMapsResourceViewController/OARequiredMapsResourceViewController.mm (hunk 1 above)
Comment (RZR-UA) [reply Sources/Controllers/Resources/OARequiredMapsResourceViewController/OARequiredMapsResourceViewController.mm]: fixed

**Pull request merged**